autogen/main/variables.tf.tmpl

```diff
@@ -147,7 +147,8 @@ variable "additional_ip_range_pods" {
 
 variable "ip_range_services" {
   type        = string
-  description = "The _name_ of the secondary subnet range to use for services"
+  description = "The _name_ of the secondary subnet range to use for services. Omit to use default range."
```

Comment:
Can we clarify the new section to something similar to: If not provided the default `34.118.224.0/20` range will be used.